apps/sms/services/js/activity/service.js

```diff
@@ -0,0 +1,93 @@
+/* global bridge */
+
+(function(exports) {
+  'use strict';
+
+  /**
+   * Name of the service that is responsible for managing activities.
+   * @type {string}
+   */
+  const SERVICE_NAME = 'activity-service';
+
+  /**
+   * Name of the system message that is leveraged by activities.
+   * @type {string}
+   */
+  const SYSTEM_MESSAGE_NAME = 'activity';
+
+  /**
+   * Reference to active bridge service instance.
+   * @type {Service}
+   */
+  var service = null;
+
+  /**
+   * Reference to currently active activity request.
+   * @type {ActivityRequestHandler}
+   */
+  var activityRequest = null;
```

Comment:
because this will eventually live in the service worker, we can't have only one singleton for this, we need a way to track several requests.